loot_importer.user.js

```diff
@@ -126,50 +156,70 @@ function ParseLootPage() {
 
   // div#productContent table tbody tr.productOverview td.productInfo h1
   var AlbumName = document.querySelectorAll("div#productContent > table > tbody > tr.productOverview > td.productInfo > *");
-  LOGGER.debug("Album Name: ", AlbumName[0].innerText); // textContent give the space formatting and we just need the text so we use innerText
-  releasetitle = AlbumName[0].innerText;
-  LOGGER.debug("Artist Name: ", AlbumName[1].innerText);
+
   releaseartist = AlbumName[1].innerText;
+  if (releaseartist == "Various Artists") {
+    // Everything is: title(format)
+    releaseartisttitle_regex = /(.*)\((.*)\)/;
+    releaseartisttitle = AlbumName[0].innerText.match(releaseartisttitle_regex);
```

Comment:
What if match fails ? `releaseartisttitle` can be `null` (for example, if matched text is `A(B`)